### ISSUE TITLE
[Bug] Can't automatically update aur version of proton-ge-custom-bin

### ISSUE THREAD

**Issue by jimmon89** [chaotic-aur/packages#2312]
### What happens?

running `sudo pamac upgrade` results in the error
```
Preparing...
Synchronizing package databases...
Refreshing chaotic-aur.db...
Refreshing AUR...
Warning: proton-ge-custom-bin: local (1:GE_Proton7_49-1) is newer than chaotic-aur (1:GE_Proton7_45-1)
Nothing to do.
Transaction successfully finished.
```
and does not upgrade to `1:GE_Proton7_50-1` due to not being able to blacklist chaotic-aur's version of the package

### What is expected to happen?

be able to automatically update `proton-ge-custom-bin` with `sudo pamac upgrade`

### More information

my suggestion is to either update the repo with the newest package or remove the outdated one from chaotic-aur
i saw in another issue that it was decided to stop building `proton-ge-custon-bin` because of `proton-ge-custom`
however they are not the same package
games like Elite Dangerous require the steam linux runtime to run and `protoon-ge-custom` does not support it

TL;DR
just remove the outdated `proton-ge-custom-bin` from chaotic-aur
since the aur version is already compiled

**Comment by Technetium1** [comment]
You should be able to update by specifying `chaotic-aur/proton-ge-custom` and confirming replacement of `proton-ge-custom-bin`.

**Comment by jimmon89** [comment]
> You should be able to update by specifying `chaotic-aur/proton-ge-custom` and confirming replacement of `proton-ge-custom-bin`.

i'm not sure i understand what you mean.
if you're implying that `proton-ge-custom` is the same as `proton-ge-custom-bin`, i suggest you take a look at the aur pages and file lists for each
`proton-ge-custom` compiles proton-ge without a major API (Steam Linux Runtime) and causes some games to not run properly under certain circumstances.
and `proton-ge-custom-bin` never "compiles" anything, it just downloads the latest proton-ge build from the [official github](https://github.com/GloriousEggroll/proton-ge-custom/releases) and places it in `/usr/share/steam/compatibilitytools.d/proton-ge-custom/`
they are not 1:1 or direct replacements of each other by any means.

However, in the case that i misunderstood what you meant.
could you please clarify?
cheers

**Comment by dr460nf1r3** [comment]
In fact, we didn't even have it on our package list anymore 🤨 I guess that is also why it was outdated. Just removed it, so we are clear.

**Comment by jimmon89** [comment]
thank you very much, i appreciate it
i was happy to bring it to your attention, and i hope it was fairly easy to remove from chaotic-aur
cheers :)
